Cargo.toml

```diff
@@ -1,8 +1,7 @@
 [workspace]
+exclude = [ "server" ]
```

Comment:
Cargo kai subcommands recurse karte parent directories mai jab talak unhe parent Cargo.toml nahi milta. Aur since mujhe `cargo fmt` run karna tha cargo confuse hora tha ki konsa toml file consider karna hai. Ispe aur ek alternative yeh hai ki tu isko include kar, but then phir tere relative paths change hote aur cargo tera pura workspace build karta as opposed to just building the server.